### ISSUE TITLE
Commented out API's

### ISSUE THREAD

**Issue by cbrammer** [Flash3001/iOSCharts.Xamarin#6]
First off, great repo. Thank you for putting the time and effort into this binding project.

It looks like there are a quite a few area's that are commented out in the `ApiDefinition` (like [this](https://github.com/Flash3001/iOSCharts.Xamarin/blob/master/ApiDefinition.cs#L1209-L1240)). What are the reasons for that?

**Comment by Flash3001** [comment]
Thank you!

At first it was for a lack of knowledge, I was having a hard time to figure out on how to properly define interface's (C#'s) relationships. 

For instance, [BarChartDataSet](https://github.com/Flash3001/iOSCharts.Xamarin/blob/master/ApiDefinition.cs#L1036) where it inherit from BarLineScatterCandleBubbleChartDataSet and implement IChartDataSet. 

From the original code, it is a class in Swift, but for some reason (Can't remember why) I had to express it as a [Protocol] in the definition. Xamarin.iOS creates three classes for objective-c protocols, as in Name, IName and NameExtentions. So BarChartDataSet would create IBarChartDataSet, which were already defined. 

In the ApiDefinition when you want to inherit you use [BaseType(typeof())]. Using C#'s usual way, it copy methods. Things like: interface IA - interface IB : IA - class A : IA - class B : A, IB - gets a little bit messy. 

So, for all that initial mess in my understanding I tried a lot of things and in many of them I commented things out so it could work (at least compile).

When I finally figured out my biggests problems.
1 - Methods were copied to classes when using interface A : IMyInterface
2 - BarChartDataSet was actually compiled to _TtC6Charts15BarChartDataSet and so on. 
3 - Lack of swiftCore.dylib and others - Xamarin.iOS doesn't give official support for binding Swift libraries, and there's no information about it anywhere. 
(2 and 3 don't stop it from compiling, but it doesn't run).

I went back to the definition and started uncommenting things. It's reasonable that I forgot to uncoment some when I finally understood how to do it. 

Some had to stay comment because they were already defined in their base classes. 

For the lines you asked, I commented all unsafe code.

When it come to 3.0 I'll redo the ApiDefinition.
